examples/ValidationErrorMessage/ValidationErrorMessage.md

```diff
@@ -0,0 +1,5 @@
+```jsx
```

Comment:
Ideally we'd have this in the context of a form element, but this is fine for now until the text input is done.